client/components/Scenario/ContentSlide.jsx

```diff
@@ -163,10 +167,11 @@ class ContentSlide extends React.Component {
         let skipButtonContent = skipButton;
 
         if (isLastSlide) {
+            const replaceexpr = /go to next slide/i;
             skipButtonContent =
                 skipOrKeep === 'skip' ? 'Skip and finish' : 'Keep and finish';
-            skipButtonTip = skipButtonTip.replace('go to next slide', 'finish');
-            fwdButtonTip = fwdButtonTip.replace('go to next slide', 'finish');
+            skipButtonTip = skipButtonTip.replace(replaceexpr, 'finish');
+            fwdButtonTip = fwdButtonTip.replace(replaceexpr, 'finish');
```

Comment:
Non-blocking nit: Do we need to replace the text? Can't we just set a new value like `skipButtonTip = "Submit response and finish"`? Might be easier to understand at a glance.

Reply:
Yeah, this is unnecessary. 🤷🏼‍♂️ I'll fix it and push changes tonight